opentext-page.html

```diff
@@ -0,0 +1,482 @@
+                        <li><a href="#Healthcare">Healthcare</a></li>
+                        <li><a href="#Public">Public Sector</a></li>
+                        <li><a href="#Oil">Oil & Gas</a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="row">
+                <div class="column">
+                    <div a href="#Services">Services</a></div>
+                    <ul>
+                        <li><a href="#Overview">Overview</a></li>
+                        <li><a href="#Consulting">Consulting services</a></li>
+                        <li><a href="#Managed">Managed services</a></li>
+                        <li><a href="#Learning">Learning Services</a></li>
+                        <li><a href="#Cloud">Cloud managed servcies</a></li>                    
+                        <li><a href="#Optimized">Optimzed services program</a><li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="row">
+                <div class="column">
+                    <div a href="#Support">Support</a></div>
+                    <ul>
+                        <li><a href="#Overview">Overview</a></li>
+                        <li><a href="#Support">My Support</a></li>
+                        <li><a href="#Knowledge">Knowledge base</a><li>
+                        <li><a href="#Support">Open a support ticket</a><li>
+                        <li><a href="#Bill">Pay my bills</a></li>
+                        <li><a href="#Account">My account details</a></li>
+                        <li><a href="#Alerts">Alerts and advisories</a></li>
+                        <li><a href="#Training">Training</a></li>
+                        <li><a href="#Consulting">Consulting</a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="column">
+                <div class="row">
+                    <div a href="#Resources">Resources</a></div>
+                    <ul>
+                        <li><a href="#Voyage">Voyager</a></li>
+                        <li><a href="#Events">Events & webinars</a></li>
+                        <li><a href="#Blogs">Blogs</a></li>
+                        <li><a href="#Press">Press releases</a></li>
+                        <li><a href="#Customer">Customer stories</a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div class="slide-show">
+            <!-- div has two columns will add second column when learn about slideshows-->
+            <img src="#img1" alt="banner for voyager fund">
+            <h2>OpenText launches Voyager fund</h2>
+            <p>1M donation to help fight food insecurity</p>
+            <button type="submit">Learn more</button>    
```

Comment:
These three lines exist in a column, separate from image above.
Please apply this to all slides.

Reply:
Added a note in  my code will learn how to do slides when we get to css.